src/expanding-panel/index.jsx

```diff
@@ -29,7 +29,9 @@ class ExpandingPanel extends Component {
     return (
       <section className={`expanding-panel${this.isOpen() ? ' open' : ''}`}>
         <header onClick={() => this.toggle()}>
-          <h3>{ this.props.title }</h3>
+          {
+            this.props.customTitle ? this.props.customTitle : <h3>{ this.props.title }</h3>
```

Comment:
this would be better if we could determine the type by inspecting the title IMO - we do something similar in react-components I think where we check if the label is type of ReactComponent or something

Reply:
it means we wouldn't need the extra `customTitle` prop, we just see what the typeof title is, and if string wrap it in h3